### PR TITLE
Baselines and pretrained word embeddings

### PR DESCRIPTION
**Pull request by jayantk**
Implemented seq2seq and seq2tree baselines by reformatting logical forms and using a different type declaration. There are still some random crashing bugs here with executing ill-formed logical forms, but i'm working through them.

Also tried pre-trained word embeddings using GloVe. These didn't help -- in fact, they hurt by ~10 accuracy points -- but I'm not confident that it's being used properly.

### CODE REVIEW

**Comment by matt-gardner** [comment]
What are you not confident about with the word embeddings?  The initializer?
